src/rpc/RPCHelpers.cpp

```diff
@@ -476,6 +476,107 @@ traverseOwnedNodes(
     return nextCursor;
 }
 
+bool
+forEachItemAfter(
+    ripple::LedgerInfo const& lgrInfo,
+    ripple::Keylet const& root,
+    ripple::uint256 const& after,
+    std::uint64_t const hint,
+    std::uint64_t limit,
+    Context const& context,
+    std::function<bool(std::shared_ptr<ripple::SLE const> const&)> const f)
```

Comment:
Can we use `traverseOwnedNodes` for this?

Reply:
Just ran some tests - seems like it will work with a few snips.

Reply:
Done